guides/common/modules/con_monitoring-hosts-by-using-insights-in-project.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="monitoring-hosts-by-using-{insights-iop-id}"]
+= Monitoring hosts by using {insights-iop}
+
+[role="_abstract"]
+You can use {Insights} to diagnose systems and downtime related to security exploits, performance degradation, and stability failures without connecting to the {RHCloud}.
```

Comment:
This should ideally make clear that with IoP, you get Advisor and Vulnerability services. (And in the corresponding non-IoP section, it should somehow make clear that the service being discussed is Advisor and not any others.)